apps/labs/pages/api/annual-report.pdf.ts

```diff
@@ -0,0 +1,33 @@
+import type { NextRequest } from 'next/server';
+
+export const config = {
+  runtime: 'experimental-edge',
+};
+
+export default async function handler(req: NextRequest) {
+  // TODO - remove after checking the logs on Vercel
+  console.log(`
+  req.url ${req.url}
+  req.referer ${req.headers.get('referer')}
+  req.ip ${req.ip}
+  user-agent ${req.headers.get('user-agent')}
+  `);
+  await fetch('https://plausible.io/api/event', {
+    method: 'POST',
+    headers: {
+      'User-Agent': req.headers.get('user-agent'),
+      'X-Forwarded-For': req.ip,
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      domain: 'gabalafou.com',
+      name: 'pageview',
```

Comment:
I wasn't exactly sure which Plausible event to use for this. I based my decision from reading the [docs page on the Events API](https://plausible.io/docs/events-api). My take was that if I use the `pageview` event, then it will be slotted into the Plausible UI just like other pageviews. [Based on my testing](https://plausible.io/labs.quansight.org?entry_page=%2Fapi%2Fannual-report.pdf), that seems to be the case.

Reply:
With the updated [URL this seems to be the new endpoint](https://plausible.io/labs.quansight.org?entry_page=%2Fapi%2Fquansight-labs-annual-report-2022.pdf)

This seems like a reasonable event

Reply:
Slight correction. After Ralf's requested change, I think the Plausible dashboard URL should look like this:

https://plausible.io/labs.quansight.org?entry_page=%2Fannual-reports%2Fquansight-labs-annual-report-2022.pdf